Assets/Scripts/PlayerController.cs

```diff
@@ -44,8 +44,8 @@ private void Move()
         Vector3 upMovement = _forward * MoveSpeed * Time.deltaTime * Input.GetAxis("Vertical");
 
         Vector3 heading = Vector3.Normalize(rightMovement + upMovement);
-
-        transform.forward = heading;
+        if (Input.GetAxis("Horizontal") != 0 || Input.GetAxis("Vertical") != 0)
+            transform.forward = heading;
```

Comment:
Could you check whether the heading is Vector3.zero instead? If we decide to refactor our Input system, we will have to touch this as well.